sway-lib-std/src/u128.sw

```diff
@@ -312,23 +312,32 @@ impl U128 {
     ///
     /// fn foo() {
     ///     let zero_u128 = U128::from(0, 0);
-    ///     let zero_u64 = zero_u128.as_u64().unwrap();
+    ///     let zero_u64 = zero_u128.try_as_u64().unwrap();
```

Comment:
Instead of updating this doc-comment which still related to `as_u64` you should add an updated version to `try_as_u64`.

Also there's a TODO at the top of this comment that should be removed.